b2/console_tool.py

```diff
@@ -344,6 +344,25 @@ def run(self, args):
         return 0
 
 
+class GetFileInfoAndDeleteFileVersion(Command):
```

Comment:
Another option is to simply make both arguments optional in the arg parsing, and put logic in the command to figure out what's what.